cmd/utils/flags.go

```diff
@@ -1143,6 +1150,19 @@ func SetNodeConfig(ctx *cli.Context, cfg *node.Config) {
 	}
 }
 
+func setSmartCard(ctx *cli.Context, cfg *node.Config) {
+	// No support for windows, currently
+	if runtime.GOOS != "windows" {
+		// Check for the existence of the PCSCD socket file. If it exists, then
+		// the smartcard wallet will be enabled. There might still be some issues
+		// e.g. when the daemon is not running. Those issues are meant to be
+		// caught at the daemon initialization level, though.
+		if fi, err := os.Stat(ctx.GlobalString(SmartCardFlag.Name)); err == nil {
+			cfg.SmartCard = fi.Mode()&os.ModeType == os.ModeSocket
+		}
+	}
```

Comment:
Is it worth mentioning to windows-users that smart card support is disabled? I don't know, think about it, I'll leave it for you to decide.

Reply:
@holiman I have updated the command line info. Like you said this morning, let's not display a message every time we start geth. I'm also going to test it on Windows, FreeBSD and macos so it might not be limited to Linux so long anyway. I expect the PCSC daemon to behave the same on these platforms.